Source/Views/StackViewController.swift

```diff
@@ -69,4 +75,80 @@ final class StackViewController: UIViewController {
         stackView.addArrangedSubview(childViewController.view)
         childViewController.didMove(toParentViewController: self)
     }
+    
+    /// A method to access the scrollView from outside and scroll to a given target ViewController. 
```

Comment:
@joanromano Why not? The stackView is private and we want to access its scrolling functionality via code from the app. Its not complete access if that's your concern, I will then use a better description.